root/elections/Index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Layout = require('../layout');
+const {l} = require('../static/scripts/common/i18n');
+const ElectionTable = require('./ElectionTable');
+
+
+const Index = () => (
```

Comment:
(In the interest of not adding more uses of the global `$c`) can we:
* make this component accept just the `elections` as a prop
* send the elections to this component by setting the `component_props` stash item in the controller
* remove `elections` from the stash so it isn't serialized twice